### PR TITLE
Column renaming hotfix

### PR DESCRIPTION
**Pull request by axelstudios**
Production hotfix:
See #1905

### CODE REVIEW

File: seed/models/columns.py [hunk 1]
```diff
@@ -643,7 +643,8 @@ def _serialize_for_extra_data(column_value):
         try:
             with transaction.atomic():
                 # check if the new_column already exists
-                new_column = Column.objects.filter(table_name=self.table_name, column_name=new_column_name)
+                new_column = Column.objects.filter(table_name=self.table_name, column_name=new_column_name,
+                                                   organization=self.organization)
```
Comment (nllong) [inline seed/models/columns.py]: nice, so making sure that the new_column is part of the right organization!

File: seed/models/columns.py [hunk 2]
```diff
@@ -680,14 +681,14 @@ def _serialize_for_extra_data(column_value):
                 # go through the data and move it to the new field. I'm not sure yet on how long this is
                 # going to take to run, so we may have to move this to a background task
                 orig_data = STR_TO_CLASS[self.table_name].objects.filter(
-                    organization=new_column.organization,
+                    organization=self.organization,
```
Comment (nllong) [inline seed/models/columns.py]: this doesn't need to be changed, but go ahead and leave it.

File: seed/models/columns.py [hunk 3]
```diff
@@ -680,14 +681,14 @@ def _serialize_for_extra_data(column_value):
...
                     data_state=DATA_STATE_MATCHING
                 )
                 if new_column.is_extra_data:
                     if self.is_extra_data:
                         for datum in orig_data:
-                            datum.extra_data[new_column.column_name] = datum.extra_data[self.column_name]
-                            del datum.extra_data[self.column_name]
+                            datum.extra_data[new_column.column_name] = datum.extra_data.get(self.column_name, None)
```
Comment (nllong) [inline seed/models/columns.py]: much better... emtpy data would be a problem.

File: seed/models/columns.py (hunk 2 above)
Comment (axelstudios) [reply seed/models/columns.py]: Yeah, but I figured for consistency I'd go ahead and just reference `self` instead

**Pull request merged**